app/waf/src/controllers/adc.controller.ts

```diff
@@ -439,12 +382,16 @@ export class AdcController extends BaseController {
               let hostname = await bigipMgr.getHostname();
               this.logger.debug(`bigip hostname: ${hostname}`);
               await bigipMgr.getLicense();
+              await bigipMgr.getConfigsyncIp(adc.management.ipAddress);
```

Comment:
`adc.management.ipAddress` is not necessary to pass in.

Reply:
I need this parameter to filter the item in iControl REST response.

Reply:
I mean `bigipMgr.config.ipAddr` is just the value you need.

Reply:
Done